tests/unit/htdocs/class/XoopsLoadTest.php

```
@@ -0,0 +1,637 @@
+    {
+        $result = @XoopsLoad::loadFramework('nonexistentframework999');
+        $this->assertFalse($result);
+    }
+
+    /**
+     * loadFramework() triggers a user warning for non-existent framework.
+     */
+    public function testLoadFrameworkTriggersWarningForNonExistentFramework(): void
+    {
+        $warningTriggered = false;
+        $previousHandler = set_error_handler(function ($errno, $errstr) use (&$warningTriggered) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$previousHandler'. (undefined)

(UnusedLocalVariable)